### DISCUSSION TITLE
Use HTML Editor in Decoupled Orchard Core

### DISCUSSION THREAD

**Discussion by mattobox** [OrchardCMS/OrchardCore#10333] [Q&A] [answered]
Is there any way to call the Admin trumbowyg HTML editor from a decoupled project? I would like to use the same editor for both environments if possible. Any help or guidance would be appreciated.

**Comment by Piedone** [comment, marked as answer]
Do you mean on the frontend (on the admin you can use it as usually)? Not quite easily since the point with decoupled is that you're not using Orchard's display management, which would provide everything necessary for the editor. You could at least include the same JS and CSS files from the OrchardCore.Resources (since those are static files, you can just reference them directly).

**Reply by mattobox** [reply]
Yes, it would be in the decoupled environment. Thanks for your response. I’ll give that a try.

**Reply by Skrypt** [reply]
You can call BuildEditorAsync on the frontend though you will have some issues with resources that will need to be included on your frontend like @piedone says.
